backend/questionnaires/migrations/0003_initial.py

```diff
@@ -0,0 +1,257 @@
+        "Jak często powinno się robić przerwy w pracy anotacyjnej?",
+        "Jak długie powinno się robić przerwy w pracy anotacyjnej?",
+        "Co byś zmieniła/zmienił w ankiecie?"
+    ]
+    sen_questions = [
+        "Jak oceniasz jakość swojego snu ostatniej nocy?",
+        "Jak czułeś/aś się dziś rano?",
+    ]
+    stres_rano_questions = [
+        "Czuję się zestresowany, niespokojny, przytłoczony.",
+        "Czuję, że panuję nad sytuacją, dobrze sobie radzę, mam wszystko pod kontrolą.",
+    ]
+    stres_wieczor_questions = [
+        "Czuję, że wszystko mnie teraz przytłacza.",
+        "Czuję, że wszystko jest teraz nieprzewidywalne.",
+    ]
+    zdrowie_wieczor_questions = [
+        "Jak oceniasz stan swojego zdrowia?",
+    ]
+    emocje_questions = [
+        "Czy właśnie czułeś silną emocję?",
+        "Jak się czułaś: Neg-Pos",
+        "Jak się czułaś: Ospały-Bardzo pobudzony",
+    ]
+    anotacja_koniec_questions = [
+        "Jak często powinno się robić przerwy w pracy anotacyjnej?",
+        "Jak długie powinno się robić przerwy w pracy anotacyjnej?",
+        "Czy któreś pytanie/polecenie dotyczące tekstu było niezrozumiałe?",
+        "Jakimi słowami opisałbyś ten tekst (tagi, słowa kluczowe)? Dlaczego?",
+        "Jakie wrażenia/emocje/odczucia wzbudza w Tobie ten tekst? Dlaczego?",
+        "Czy tekst wzbudza w Tobie jakiekolwiek emocje? Dlaczego?",
+        "etc? Dlaczego?",
+        "Tekstów było?"
+        "Jaka liczba tekstów byłaby odpowiednia?",
+        "Co byś zmieniła/zmienił w badaniu?",
+    ]
+    ankieta_tygodniu_questions = [
+        "Czy spotkanie na Google Meets było problematyczne?",
+        "Czy wolałabyś/wolałbyś spotkanie na Zoom?",
+        "Czy wolałabyś/wolałbyś spotkanie na innej platformie (jakiej)?",
+        "Czy korzystałeś/korzystałaś z kontaktu z osobą odpowiedzialną za rozwiązywanie problemów i wsparcie?",
+        "Czy masz jakieś pomysły/sugestie dotyczące webinarium?",
+        "Jak często powinno się robić przerwy w pracy anotacyjnej?",
+        "Jak długie powinno się robić przerwy w pracy anotacyjnej?",
+        "Czy któreś pytanie/polecenie dotyczące tekstu było niezrozumiałe?",
+        "Jakimi słowami opisałbyś ten tekst (tagi, słowa kluczowe)? Dlaczego?",
+        "Jakie wrażenia/emocje/odczucia wzbudza w Tobie ten tekst? Dlaczego?",
+        "Czy tekst wzbudza w Tobie jakiekolwiek emocje? Dlaczego?",
+        "etc? Dlaczego?",
+        "Tekstów było?"
+        "Jaka liczba tekstów byłaby odpowiednia?",
+        "Co byś zmieniła/zmienił w badaniu?",
+    ]
+    questions = ipip_bfm_questions + demografia_questions +  humor_styles_questions + spane_questions + skala_questions + phq9_questions + pss_questions + zdrowie_questions + ress_ema_questions + paq_questions + zwrotna_questions + sen_questions + stres_rano_questions + stres_wieczor_questions + zdrowie_wieczor_questions + emocje_questions + anotacja_koniec_questions + ankieta_tygodniu_questions
```

Comment:
Hello! I am sorry I missed this, but I just found out that you missed adding `swlsa_questions` to the questions on line 229 here. So the questions for questionnaire swlsa are not available XD

Reply:
I have updated it XD thank you

Reply:
thank you! it should be fine now XD